CMakeLists.txt

```diff
@@ -174,7 +174,7 @@ else()
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -W -Wextra -Wall")
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wsign-compare -Wshadow -Wno-unused-parameter -Wno-unused-variable -Woverloaded-virtual -Wnon-virtual-dtor -Wno-missing-field-initializers -Wno-strict-aliasing")
   if(MINGW)
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-format")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-format -fno-asynchronous-unwind-tables")
```

Comment:
Curious to know if `-fno-asynchronous-unwind-tables` is intentional here?

Reply:
Yeah, the ci `JOB_NAME=cmake-mingw` will fail without this option. Plz refer to the [comment](https://github.com/facebook/rocksdb/pull/6098#issuecomment-561232056) above. BTW, I'm not a windows user. LoL

Reply:
Looks like this is flaky, but has been reported at https://sourceforge.net/p/mingw-w64/mailman/message/36287627/ and a few other places.

Reply:
Yeah, that's seems a bug of `mingw`, I have to add it in order to pass the ci checks.